blue_helpers/package.xml

```diff
@@ -1,8 +1,7 @@
-<?xml version="1.0"?>
-<package>
+<package format="2">
   <name>blue_helpers</name>
   <version>0.0.0</version>
-  <description>The blue_helpers metapackage contains helpful tools for running the Blue robot arm</description>
+  <description>The blue_helpers package contains helpful tools for the Blue robot arm</description>
 
   <maintainer email="phil80301@berkeley.edu">Philipp Wu</maintainer>
```

Comment:
Your welcome to change the maintainer and author to yourself @methylDragon. It's your contribution!

Reply:
Oh! Alrighty then, cheers!